packages/next/src/build/utils.ts

```diff
@@ -2026,13 +2026,6 @@ const dir = path.join(__dirname)
 process.env.NODE_ENV = 'production'
 process.chdir(__dirname)
 
-// Make sure commands gracefully respect termination signals (e.g. from Docker)
-// Allow the graceful termination to be manually configurable
```

Comment:
can we move this comment to next server?

Reply:
sure - I just moved them to `start-server.ts` just before the process.env check.

Also added a Log.Info to let users know the cleanup process started in case they wonder why it's not shutting down immediately.